akka-http-core/src/main/scala/akka/http/Http.scala

```diff
@@ -145,7 +144,7 @@ class HttpExt(config: Config)(implicit system: ActorSystem) extends akka.actor.E
     val remoteAddr = new InetSocketAddress(host, port)
     val layer = clientLayer(remoteAddr, settings, log)
 
-    val transportFlow = StreamTcp().outgoingConnection(remoteAddr, localAddress,
+    val transportFlow = Tcp().outgoingConnection(remoteAddr, localAddress,
       options, settings.connectingTimeout, settings.idleTimeout)
```

Comment:
@jrudolph @sirthias heads up for eventual merge conflicts: small rename of StreamTcp to Tcp and `interface` and `port` params instead of InetSocketAddress.